schema/deploy/function_application_application_revision_status.sql

```diff
@@ -0,0 +1,25 @@
+-- Deploy ggircs-portal:function_application_application_revision_status to pg
+-- requires: table_application_revision_status
+-- requires: table_application
+
+begin;
+
+create or replace function ggircs_portal.application_application_revision_status(application ggircs_portal.application)
```

Comment:
At the moment, I'm using the one directly on Application as the latest status for the application, regardless of draft/submitted. I think this will probably get refactored out when we refactor the fragments from Application -> ApplicationRevision